ui/components/component-library/avatar-base/avatar-base.js

```diff
@@ -3,21 +3,26 @@ import PropTypes from 'prop-types';
 import classnames from 'classnames';
 
 import Box from '../../ui/box/box';
-import { COLORS, SIZES } from '../../../helpers/constants/design-system';
+import {
+  COLORS,
+  SIZES,
+  BACKGROUND_COLORS,
+  BORDER_COLORS,
+} from '../../../helpers/constants/design-system';
 
-export const BaseAvatar = ({
+export const AvatarBase = ({
   size = SIZES.MD,
   children,
-  backgroundColor = COLORS.BACKGROUND_ALTERNATIVE,
-  borderColor = COLORS.BORDER_DEFAULT,
+  backgroundColor = BACKGROUND_COLORS.BACKGROUND_ALTERNATIVE,
+  borderColor = BORDER_COLORS.BORDER_DEFAULT,
```

Comment:
I think the use of `COLORS` was correct, although we have specific conts we should just be using them for propTypes and to limit the options of the controls in storybook. For all other instances we should use the general object